variables.tf

```
@@ -39,7 +39,7 @@ variable "ecs_subnet_ids" {
 }
 
 variable "environment" {
-  type        = map
+  type        = map(any)
```

Would this not be a better fit perhaps?

```suggestion
  type        = map(string)
```

---

TBH: Not sure, but that might be ok.

Basically this is the change as applied by `terraform fmt` due to new versions..